File: Scala/day8.scala
object Solution {
    def main(args: Array[String]) {
        var phoneBook:Map[String, Int] = Map()
        for (_ <- 1 to readInt) {
           phoneBook += (readLine-> readInt)
        }
        var read = true
        while (read) {
            val name = readLine
            read = name != null
            if(read)
            println(if(phoneBook.contains(name)) s"$name="+phoneBook(name) else "Not found")
        }
    }
}

File: SWIFT/day8.swift

var noOfContacts = Int(readLine()!)!
var phoneBook:[String:Int]=[:]

for _ in 0..<noOfContacts{
phoneBook[readLine()!] = Int(readLine()!)!
}

while let key = readLine() {
  if let foundKeyValue = phoneBook[key] {
      print("\(key)=\(phoneBook[key]!)")
    } else {
      print("Not found")
  }
}
